node/core/runtime-api/src/lib.rs

```diff
@@ -270,7 +270,7 @@ where
 				)
 			}
 		} else {
-			self.spawn_handle.spawn_blocking(API_REQUEST_TASK_NAME, request);
+			self.spawn_handle.spawn_blocking(API_REQUEST_TASK_NAME, "", request);
```

Comment:
Not keen on using empty strings, in the ui this might be confusing.

Reply:
I think this should work fine at least for Grafana - we group all tasks that have no group name. Not sure in what situation would this be confusing.

Reply:
Depends on how you look at it, i.e. filtering for no-subsystem might require escaping in loki, legend names in grafana graphs show nothing, I'd prefer to have a default `NONE` value or similar. Just a nit.

Reply:
I agree, having a default bucket to group "ungrouped" tasks gives consistency. The first version of the substrate PR provided default values, but the feedback suggested empty strings make more sense. I didn't consider at the time the extra pains to make an argument. That said, for this to make sense it has to apply to substrate as well, so I'll have to get buy in on these changes there first.

Reply:
fixed, group name is now optional and a default is used if tasks are spawned with group None .